pulsar-functions/instance/src/main/python/python_instance.py

```diff
@@ -309,15 +303,12 @@ def get_metrics(self):
     metrics_data = InstanceCommunication_pb2.MetricsData()
     # total metrics
     metrics_data.receivedTotal = int(total_received) if sys.version_info.major >= 3 else long(total_received)
-    metrics_data.processedTotal = int(total_processed) if sys.version_info.major >= 3 else long(total_processed)
     metrics_data.processedSuccessfullyTotal = int(total_processed_successfully) if sys.version_info.major >= 3 else long(total_processed_successfully)
     metrics_data.systemExceptionsTotal = int(total_sys_exceptions) if sys.version_info.major >= 3 else long(total_sys_exceptions)
     metrics_data.userExceptionsTotal = int(total_user_exceptions) if sys.version_info.major >= 3 else long(total_user_exceptions)
     metrics_data.avgProcessLatency = avg_process_latency_ms
     metrics_data.lastInvocation = int(last_invocation) if sys.version_info.major >= 3 else long(last_invocation)
     # 1min metrics
-    metrics_data.receivedTotal_1min = int(total_received_1min) if sys.version_info.major >= 3 else long(total_received_1min)
```

Comment:
why are we removing receivedTotal_1min ?

Reply:
Accidentally removed. Restored back